assets/component-cart-items.css

```
@@ -89,12 +91,18 @@
 }
 
 cart-remove-button .button {
+  box-shadow: none;
+  color: rgb(var(--color-base-text));
```

```suggestion
  color: rgb(var(--color-base-text),.75);
```
Will need to have a normal and hover state, just like the link, I believe the opacity is set to 75% and the hover is at 100%. Correct?

---

Correct, hover is set at 100%